ClinGen/clincoded
# Connects to #925. Adding activity indicator.

**Technical details:**
1. Added a stateless activity indicator method that can be re-used, along with its own CSS module.
2. The downside of this activity indicator implementation is that it requires the parent DOM element (new or existing) to have `position: relative;` CSS property.
3. The activity indicator has been applied to the tables that are dependent on fetched data for the Basic Info, Population, Predictors, and Gene-centric tabs.

**Steps to test:**
1. Login and use **17662** and/or **21134** variant_id at the variation selection interface.
2. Upon being redirected to the Basic Info tab, expect to see the display of the activity indicators in each table until the data starts being rendered. Particularly for the ClinVar Interpretations table, the "No data was found..." message shouldn't be visible prior to the rendering of RCV data.
3. Proceed to other tabs and refresh the page. Expect to see the consistent behavior of the activity indicator throughout different tables.
4. For testing a variant without any RCVs, try **CA501097** and expect to see the display of the "No data was found..." message in the ClinVar Interpretations table on the Basic Info tab.
